mitodl/mitxpro
# Added bulk assignment CSV download to bulk coupon form

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1121 

#### What's this PR do?
Allows an admin user to download a CSV of coupons that were assigned to users in the bulk coupon form

#### How should this be manually tested?
- Create a simple single-column CSV containing valid user email addresses
- Go to the bulk coupon form (`/ecommerce/admin/enroll/`), upload that CSV, and choose a course run and coupon
- Submit the form, then click the download link that appears after the request successfully responds

That downloaded CSV should contain all of the emails in your original CSV along with enrollment links and coupon codes for the coupons assigned to them

#### Screenshots (if appropriate)
<img width="609" alt="ss 2019-09-25 at 13 10 52 " src="https://user-images.githubusercontent.com/14932219/65624344-a7d01b00-df97-11e9-9e67-097c94105c45.png">
<img width="1237" alt="ss 2019-09-25 at 13 24 00 " src="https://user-images.githubusercontent.com/14932219/65624388-c1716280-df97-11e9-9eb1-0d99fcb58c1b.png">
